getting-started/blazor/apply-different-themes-using-blazor.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Apply Different Themes using Blazor | FusionCharts
+description: This article will showcase applying different themes to the chart at runtime.
+heading: Apply Different Themes using Blazor
+---
+
+In FusionCharts Suite XT, you can manually set each chart's cosmetics and functional attributes in the corresponding JSON/XML file. This works when dealing with only a small number of charts. As the number of charts increases, so do the hassles. FusionCharts Suite ships with predefined themes that you can use to set your chart's visual appearance or behavior.
+
+FusionCharts Suite XT ships with the following predefined themes:
+
+* `fusion` (since v3.13.0)
+* `gammel` (since v3.13.0)
+* `candy` (since v3.13.0)
+* `zune`
+* `ocean`
+* `carbon`
+
+This article focuses on how to apply different themes to a chart at runtime using the `blazor-fusioncharts` component. 
+
+The following is an example of a chart configured to change the theme. 
+
+[MISSING IMAGE](/images/)
+
+The complete code of the above sample is given below:
+
+```javascript
+missing-blazor-code
```

Comment:
is this intentional?

Reply:
yes, it was intentional. I was missing a code sample until this morning.

Reply:
This was added and is updated in the latest PR